### ISSUE TITLE
Next video is played even if "playlist autoplay" button is disabled

### ISSUE THREAD

**Issue by eight04** [ParticleCore/Particle#464]
Browser Name: Firefox

Browser Version: 53.0a2

Operating System: Windows 7

Software Type: Userscript

Software Version: 1.7.9

Userscript Manager: Greasemonkey

Userscript Version: 3.10

Problem Description: 
While disabling the autoplay option in the playlist, Youtube still autoplays next video if

1. Any video on the playlist is played.
2. In fullscreen.

Steps to reproduce: 
1. Go to https://www.youtube.com/watch?v=T66-dT9j12E&list=PL0bHKk6wuUGL_Qd34mf0XsQnyiDk2OeGR&index=31.
2. Disable playlist autoplay.
3. Click any item on the playlist and wait the video to load.
4. Go fullscreen.
5. After the video ends, next video is played.

**Comment by mb1234** [comment]
Try this code as a greasemonkey script. It works for me.

```// ==UserScript==
// @name        Disable Youtube Autoplay
// @namespace   uncheck_boxes
// @version     1
// @grant       none
// ==/UserScript==

function f() {
 try {
 var node = document.getElementById('autoplay-checkbox').parentNode.parentNode;
 if (node.className == 'checkbox-on-off')
 node.parentNode.removeChild(node);
 }
 catch (e) {}
}
f();
document.body.addEventListener('DOMSubtreeModified', f, false);```

**Comment by ParticleCore** [comment]
@mb1234 That's for autoplay up-next, which is not related to this issue.

**Comment by ParticleCore** [comment]
@eight04 I have released version 1.8.0 with a fix for this issue, let me know if it works correctly for you as well.

**Comment by eight04** [comment]
Yes it works! Thanks for the quick fix!
